spec/requests/needs_spec.rb

```diff
@@ -56,7 +62,7 @@
   describe '#update' do
     context 'success' do
       it 'redirects to the need' do
-        expect(Services::SendNeedNotifications).to receive(:call)
+        expect(Services::NeedNotifications::Update).to receive(:call).and_return(true)
```

Comment:
fwiw you can strengthen these expectations by declaring how many times you expect the stubbed method to be called, eg. 
```
expect(Services::NeedNotifications::Update).to receive(:call).and_return(true).exactly(2).times
```